### DISCUSSION TITLE
[Duvida] Curso web moderno da Udemy

### DISCUSSION THREAD

**Discussion by MaiconJordan** [frontendbr/forum#1745] [Discussão]
Caros amigos, pra iniciar os estudos e na carreira queria um curso que me desse uma base boa, alguns indicam esse curso da Udemy. Alguém pode me dá umas dicas se vale a pena, ele é bem barato, apenas 19,00. Então pela grade de assuntos nele, será que posso aprender o suficiente pra fazer algo que dê pra agregar no meu portfólio? 

O curso seria esse : https://www.udemy.com/course/curso-web/

**Comment by MrS4w** [comment]
Você vai aprender a base de tudo com esse curso aí, não completei nem 60% dele e consegui trabalhar com JS a partir daí, mas acho que é válido para quem está no começo, se você já tiver alguma noção vai achar muito massante, outro ponto também é que você pode comprar e testar o curso, se não é o que você espera, você pode pedir reembolso.

**Comment by MaiconJordan** [comment]
Entendi, vc disse que começou a trabalhar com JS, mas no sentindo que vc conseguiu uma vaga com a base que vc teve desde curso?

**Comment by MrS4w** [comment]
Sim, consegui com conteúdos que vi nesse curso que na época se chamava "Curso Web Moderno 2018" o instrutor foi esperto em mudar só o título ano passado e esse ano hahaha.

**Comment by wilsonneto-swe** [comment]
@MaiconJordan dois cursos que fize que me ajudaram demais, se aceitar indicações, foram:

- GoStack da Rocketseat (inscrições fechadas nesse momento)

- DevPleno - ReactJS ([http://bit.ly/reactjs-dev-pleno](http://bit.ly/reactjs-dev-pleno))

O curso do devpleno teve um custo benefício bem melhor, comecei com ele focando em ReactJS para web, já me deu a base para passar de backend para fullstack, dai com a valorização que ele me deu consegui comprar o GoStack que me deu uma base bem legal de fullstack JS também além de mais conhecimentos sobre react js... Para ser sincero a valorização que tive no mercado foi maior q o valor dos dois cursos juntos por mês, o importante é disciplina e sempre estar melhorando. Veja cursos como investimento, e terá retorno...

Forte abraço!

----

Wilson Gomes
Linkedin: [https://www.linkedin.com/in/wilsonnetobr/](https://www.linkedin.com/in/wilsonnetobr/)
